esx_service/utils/auth_data_test.py

```diff
@@ -435,6 +444,7 @@ def test_remove_tenants(self):
                                                           description='Some tenant',
                                                           vms=vms,
                                                           privileges=privileges)
+
```

Comment:
It seems this file is having only empty newline, please remove this file from the review if it is an accidental change.